CHANGELOG.md

```diff
@@ -1,3 +1,24 @@
+## [4.3.3](https://github.com/adobe/helix-status/compare/v4.3.2...v4.3.3) (2019-09-04)
```

Comment:
This does not belong to the PR, this file is auto-generated.

Reply:
I think I based my changes on the wrong branch…none of those are changes contained in the two commits I actually created…

Reply:
There, I rebased and this seems solved…